projects/ngneat/dialog/src/lib/dialog.component.ts

```diff
@@ -74,7 +74,7 @@ export class DialogComponent implements OnInit, OnDestroy {
     this.nodes.forEach(node => host.appendChild(node));
 
     if (config.windowClass) {
-      host.classList.add(config.windowClass);
+      host.classList.add(...config.windowClass.split(/\s/).filter(x => x));
```

Comment:
This will work in IE?

Reply:
Well, no and yes. It doesn't work natively. IE isn't supported by project's [browserlist](https://github.com/ngneat/dialog/blob/master/browserslist). If you like it work's with IE change the browserlist and required ployfills should be added automatically. Still works (and tested via [Virtual Machines](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)) with Edge 18 (supported by the browserlist)

Reply:
Let's change it to:

```ts
const classNames = config.windowClass.split(/\s/).filter(x => x);
classNames.forEach(name => host.classList.add(name))
```

Reply:
Done, tested and pushed.